File: day01Scala/src/main/scala/com/sjyttkl/chapter07/javapackage/TestTiger.java
package com.sjyttkl.chapter07.javapackage;

public class TestTiger {
    public static void main(String[] args) {
        //使用xm的Tiger
        com.sjyttkl.chapter07.javapackage.xm.Tiger tiger01 = new com.sjyttkl.chapter07.javapackage.xm.Tiger();
        //使用xh的Tiger
        com.sjyttkl.chapter07.javapackage.xh.Tiger tiger02 = new com.sjyttkl.chapter07.javapackage.xh.Tiger();

        System.out.println("tiger01=" + tiger01 + "tiger02=" + tiger02);
    }
}

File: src/main/scala/com/sjyttkl/chapter07/scalapackage/TestTiger.scala
package com.sjyttkl.chapter07.scalapackage

object TestTiger {
  def main(args: Array[String]): Unit = {
    //使用xh的Tiger
    val tiger1 = new com.sjyttkl.chapter07.scalapackage.xh.Tiger
    //使用xm的Tiger
    val tiger2 = new com.sjyttkl.chapter07.scalapackage.xm.Tiger
    println(tiger1 + " " + tiger2)

  }
}

class Employee {

}
